### ISSUE TITLE
Example how to read X9 files

### ISSUE THREAD

**Issue by herbertpimentel** [TheLastCheck/tlc.open.java#8]
Please can you provide a example of how to read the X9 Files and extract information from it ?!

**Comment by jerrybowman** [comment]
Yes. I have a sample of how to create a file here: https://thelastcheck.github.io/#/tlc-io-x937

I'll add a section on reading and post it shortly.
